deployment/src/main/java/io/quarkiverse/loggingui/quarkus/logging/ui/deployment/LoggingUiConfig.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkiverse.loggingui.quarkus.logging.ui.deployment;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot
+public class LoggingUiConfig {
+
+    /**
+     * The base path, defaults to /loggers
+     */
+    @ConfigItem(defaultValue = "/loggers")
+    String basePath;
```

Comment:
This will cause the path to be configured at build time, right? I want to enable users to change this at runtime (start time, actually). Imagine that in one environment you have a fixed path to set your non-app endpoints, or that you have heterogeneous environments where you need the same app to provide such functionalities in different endpoints.

Reply:
Ok, we can make it a Runtime config. I'll change it

Reply:
Maybe explain a bit more what you want with examples. I am not sure I follow.

Reply:
Thinking about this a bit more. If you can change the path during runtime, you can bypass any security added. Not sure if this is a good idea.

Reply:
Yes, you could. But given that it would require a change in the application.yml file, I think it is safe to say that only trusted people would do that, therefore I think that bypassing the security is not a concern.
It feels strange to me that I have to rebuild my application in order to make environment specific configuration. When I say "different environments" I do not mean dev/staging/production, but in my current setup we have different production clusters where our apps run, therefore we would have different "production environments". Sometimes they even have their own staging and dev.

This feels much more like a configuration that could be changed anytime as needed (like [client configurations](https://quarkus.io/guides/rest-client#create-the-configuration) or the server port) than something that should be defined at build time, since it is specific to the environment it is deployed and not to the app itself.

Reply:
Ok, let's say we make basePath changeble on Runtime, that means I can, in runtime change the path where the logger-ui endpoint would be published. The security is setup in application properties to secure a certain path, that I just now changed in runtime. 

Also, w.r.t environments, we are talking about the path, I am not sure why that needs to change between environments ? Different environments will already have different  host and/or ports, so what value does also changing the path brings ?

Reply:
I get the security argument, but the risk also exists at build time configuration. And considering that only trusted people should have write access to the configuration, I do not see that as critical.

You might have one scrapper that runs on /metrics, others that runs on /prometheus, for instance.